.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "SDL"]
```

Comment:
already discussed in #466, not the solution we'd like to go for

Reply:
got this working i think! great find earlier.